Added custom event handler section (#3853)
andymccurdy/redis-py: docs/multi_database.rst

    )
    client = MultiDBClient(cfg)

Custom failover callbacks
-------------------------

> You may want to make a custom actions when failover happens. For example, you may want to collect some metrics,

Reviewer: ```suggestion
You may want to activate custom actions when failover happens. For example, you may want to collect some metrics,
```

logs or externally persist a connection state.

> You can register your own event listener for `ActiveDatabaseChanged` event that was emitted when failover happened, via

Reviewer: ```suggestion
You can register your own event listener for the `ActiveDatabaseChanged` event (which is emitted when a failover happens) using
```
